frontend/src/setupProxy.js

```diff
@@ -4,7 +4,7 @@ module.exports = function(app) {
   app.use(
     '/api',
     createProxyMiddleware({
-      target: process.env.API_URL || 'http://backend:8080/',
+      target: process.env.API_URL || 'http://localhost:5000/',
```

Comment:
This was most likely going nowhere. Thoughts @asanchezr @devinleighsmith?

Reply:
this was going to the backend API docker container when we used to run the API in docker locally. We don't anymore so I'm okay with this change. You should have API_URL defined in your env anyways